### PR TITLE
Feature evaluation

### PR DESCRIPTION
**Pull request by crispaulrubi**
Integrated subject page for students

When pressing evaluate button in dashboard, page redirects to subjects_taken page
pressing evaluate will redirect to the eval sheet

### CODE REVIEW

**Comment by rosalierazonable** [comment]
- FINDINGS

> Integrated subject page for students

- can we increase the font sizes as status and the faculty name? 
- I think nice if implement siya sa backend and naka sentence case ang name sa subject daan para capital jud and una sa letter bisan pa ug masayup enter sa database.
- Status of the subject that hasn't been evaluated yet is **null**? 
- Error when clicking the evaluate button under the subject with status null: 

```
mysqli_sql_exception #1054
Unknown column 'eval_sheet_id' in 'where clause' 
```

> When pressing evaluate button in the dashboard, the page redirects to the subjects_taken page

- successful, works as expected sa iya behavior

> pressing evaluate will redirect to the eval sheet

- it doesn't redirect to the eval sheet due to the error mentioned above

File: app/Controllers/Evaluation.php [hunk 1]
```diff
@@ -5,59 +5,329 @@
...
+  /**
+   * Get Previous Answers of user
+   */
+  protected function getPreviousAnswers($eval_sheet_id)
+  {
+    $evalAnswersModel = new EvalAnswersModel();
+    $prevAnswers = $evalAnswersModel
+      ->where('user_id', $_SESSION['logged_user']['id'])
+      ->where('is_deleted', 0)
+      ->where('eval_sheet_id', $eval_sheet_id)
```
Comment (rosalierazonable) [inline app/Controllers/Evaluation.php:135]: nag add ka ug eval_sheet_id attribute sa database?

**Review by rosalierazonable — CHANGES_REQUESTED** [review]
# FINDINGS

> Integrated subject page for students

- can we increase the font sizes as status and the faculty name? 
- I think nice if implement siya sa backend and naka sentence case ang name sa subject daan para capital jud and una sa letter bisan pa ug masayup enter sa database.
- Status of the subject that hasn't been evaluated yet is **null**? 
- Error when clicking the evaluate button under the subject with status null: 

```
mysqli_sql_exception #1054
Unknown column 'eval_sheet_id' in 'where clause' 
```

> When pressing evaluate button in the dashboard, the page redirects to the subjects_taken page

- successful, works as expected sa iya behavior but, 
- button misalignment sa evaluation sheet wala pa na address
- When the evaluation for a particular subject is already set to "Completed" and you click the evaluate again, it still lets you make an evaluation for that specific subject then back to square one. Murag need na i-implement sa na siya

> pressing evaluate will redirect to the eval sheet

- it doesn't redirect to the eval sheet due to the error mentioned above

File: app/Controllers/Evaluation.php [hunk 2]
```diff
@@ -5,59 +5,329 @@
...
 
-  public function set_category($evalSheetId = null) {
-    $css = ['custom/modalAddition.css', 'custom/alert.css'];
-    $js = ['custom/alert.js'];
-
+  public function evaluate($eval_sheet_id = null)
+  {
     $data = [];
+    
+    if ($this->request->getMethod() === 'post') {
```
Comment (rosalierazonable) [inline app/Controllers/Evaluation.php:55]: - Checking if ang status of that **eval_sheet is already "completed"** then dapat di na siya maka himo ug changes for that subject na evaluation
- Default status is **"Open" or "open for evaluation"**, not **null**

**Review by rosalierazonable — CHANGES_REQUESTED** [review]

File: app/Controllers/Evaluation.php (hunk 1 above)
Comment (crispaulrubi) [reply app/Controllers/Evaluation.php:135]: oo nag add ko, nakalimot kog export sa db

**Pull request merged**